astropy/coordinates/baseframe.py

```diff
@@ -156,6 +157,67 @@ def __new__(mcls, name, bases, members):
                 'Could not find all expected BaseCoordinateFrame class '
                 'attributes.  Are you mis-using FrameMeta?')
 
+        # Unless overridden, velocity name defaults are:
```

Comment:
This text should be in the `BaseCoordinateFrame` docstring somewhere. ( I realize it's awkward to put it there even though the implementation is here, but c'est la vie.  Perhaps note in this comment that it's also there?)